patchwork-fml/src/main/java/net/minecraftforge/fml/client/event/ConfigChangedEvent.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2019, 2019
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.fml.client.event;
+
+import javax.annotation.Nullable;
+
+import net.minecraftforge.eventbus.api.Event;
+
+/**
```

Comment:
Since `GuiConfig` doesn't exist in Forge (and sounds like an MCP name), it would be preferable to replace it with simply `config screen` instead.